packages/r/reveal.js.json

```
@@ -1,8 +1,8 @@
 {
   "name": "reveal.js",
-  "filename": "js/reveal.min.js",
+  "filename": "dist/reveal.js",
```

```suggestion
  "filename": "reveal.js",
```

As we're using `basePath`, these will be pulled up and there won't be a `dist` subdirectory.